<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Statistical physics approaches to community ecology | Robert Marsland</title>
<meta name="generator" content="Jekyll v4.0.0" />
<meta property="og:title" content="Statistical physics approaches to community ecology" />
<meta name="author" content="Robert A Marsland III" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Microbial ecosystems manifest one of the main technical obstacles to predictive modeling of biological systems, which arises from their extreme heterogeneity. The standard methods of statistical mechanics produce powerful and robust predictions about the behavior of large collections of functionally identical particles, which are easily extended to mixtures of a few different kinds of components. But in most biological systems, the number of distinct types of components is unmanageably large, whether we are considering the 20,000 different genes in the human genome, or the 20,000 distinct metabolites involved in biochemical networks. Microbial ecosystems reproduce this same difficulty on a bigger length scale, with hundreds to thousands of coexisting “species” in a given natural community. The development of inexpensive DNA sequencing has made these systems an attractive setting for developing and testing new theoretical methods for overcoming this hurdle. Highly diverse microbial communities can be readily found on any plant leaf, pond droplet, or grain of soil, and sequencing technology makes it possible to determine which species are present there and in what abundance." />
<meta property="og:description" content="Microbial ecosystems manifest one of the main technical obstacles to predictive modeling of biological systems, which arises from their extreme heterogeneity. The standard methods of statistical mechanics produce powerful and robust predictions about the behavior of large collections of functionally identical particles, which are easily extended to mixtures of a few different kinds of components. But in most biological systems, the number of distinct types of components is unmanageably large, whether we are considering the 20,000 different genes in the human genome, or the 20,000 distinct metabolites involved in biochemical networks. Microbial ecosystems reproduce this same difficulty on a bigger length scale, with hundreds to thousands of coexisting “species” in a given natural community. The development of inexpensive DNA sequencing has made these systems an attractive setting for developing and testing new theoretical methods for overcoming this hurdle. Highly diverse microbial communities can be readily found on any plant leaf, pond droplet, or grain of soil, and sequencing technology makes it possible to determine which species are present there and in what abundance." />
<link rel="canonical" href="/research/typical-ecology" />
<meta property="og:url" content="/research/typical-ecology" />
<meta property="og:site_name" content="Robert Marsland" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2019-08-21T02:00:06+02:00" />
<script type="application/ld+json">
{"url":"/research/typical-ecology","headline":"Statistical physics approaches to community ecology","datePublished":"2019-08-21T02:00:06+02:00","dateModified":"2019-08-21T02:00:06+02:00","mainEntityOfPage":{"@type":"WebPage","@id":"/research/typical-ecology"},"author":{"@type":"Person","name":"Robert A Marsland III"},"description":"Microbial ecosystems manifest one of the main technical obstacles to predictive modeling of biological systems, which arises from their extreme heterogeneity. The standard methods of statistical mechanics produce powerful and robust predictions about the behavior of large collections of functionally identical particles, which are easily extended to mixtures of a few different kinds of components. But in most biological systems, the number of distinct types of components is unmanageably large, whether we are considering the 20,000 different genes in the human genome, or the 20,000 distinct metabolites involved in biochemical networks. Microbial ecosystems reproduce this same difficulty on a bigger length scale, with hundreds to thousands of coexisting “species” in a given natural community. The development of inexpensive DNA sequencing has made these systems an attractive setting for developing and testing new theoretical methods for overcoming this hurdle. Highly diverse microbial communities can be readily found on any plant leaf, pond droplet, or grain of soil, and sequencing technology makes it possible to determine which species are present there and in what abundance.","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="/feed.xml" title="Robert Marsland" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Robert Marsland</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/research/">Research</a><a class="page-link" href="/CV/">CV</a><a class="page-link" href="/publications/">Publications</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Statistical physics approaches to community ecology</h1>
    <p class="post-meta"></p>
      <img class="feat-img" src="/assets/images/Fig1_scheme_trimmed-01.jpg" alt="Random Interactions" />
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p>Microbial ecosystems manifest one of the main technical obstacles to predictive modeling of biological systems, which arises from their extreme heterogeneity. The standard methods of statistical mechanics produce powerful and robust predictions about the behavior of large collections of functionally identical particles, which are easily extended to mixtures of a few different kinds of components. But in most biological systems, the number of distinct types of components is unmanageably large, whether we are considering the <a href="https://en.wikipedia.org/wiki/Human_genome">20,000 different genes in the human genome</a>, or the <a href="https://www.genome.jp/kegg/docs/statistics.html">20,000 distinct metabolites</a> involved in biochemical networks. Microbial ecosystems reproduce this same difficulty on a bigger length scale, with hundreds to thousands of coexisting “species” in a given natural community. The development of inexpensive DNA sequencing has made these systems an attractive setting for developing and testing new theoretical methods for overcoming this hurdle. Highly diverse microbial communities can be readily found on any plant leaf, pond droplet, or grain of soil, and sequencing technology makes it possible to determine which species are present there and in what abundance.</p>

<p>My colleagues and I have been adapting <a href="https://iopscience.iop.org/article/10.1088/1742-5468/aab04e/meta">methods from the physics of disordered materials</a> to this new setting, deriving predictions about the diversity, composition and sensitivity of heterogeneous communities based on summary statistics about consumption preferences and byproduct secretion in the regional species pool. The key assumption behind these methods is that the parameters characterizing each species can be modeled as independent random variables drawn from a given probability distribution. This seems like an extreme assumption, given how much specific structure is already known about microbial behavior. But we have shown that at sufficiently high levels of diversity, the strong interactions between specific pairs of organisms based on known mechanisms can be overwhelmed by the sheer number of weak, uncharacterized interactions with all the other species in the system <a href="#cui2019diverse">(Cui, Marsland III, &amp; Mehta, Preprints)</a>.</p>

<p>To check our calculations and facilitate hypothesis generation, I have also been developing <a href="https://github.com/Emergent-Behaviors-in-Biology/community-simulator">a Python package</a> for rapidly simulating diverse microbial ecosystems with randomly sampled parameters <a href="#marsland2019community">(Marsland III, Cui, Goldford, &amp; Mehta, 2020)</a>. Using this package, I have shown that the robust patterns observed in large-scale surveys of microbial diversity (such as the Human Microbiome Project and Earth Microbiome Project) reflect the “typical” structure of diverse communities <a href="#marsland2019minimal">(Marsland III, Cui, &amp; Mehta, 2020)</a>.</p>

<p>I have also used this software to study some novel ecological phenomena that can arise in self-sufficient microbial ecosystems <a href="#marsland2019available">(Marsland III et al., 2019)</a>. Since microbes generically produce usable byproducts whenever they consume an energy source, they can autonomously generate a complex chemical environment that supports a diverse community even when only a single energy source is supplied externally. This differs from the standard case considered in macroscopic ecology, where environmental conditions are either fixed or modified only through consumption. By performing simulations of this extreme case of a fully self-generated environment, I identified a transition in community structure as a function of the supply rate of the externally provided resource. At high supply rates, the community-level metabolic network percolates, and all possible byproducts are produced at sufficiently high concentrations to significantly contribute to the growth of some set of species. The scaling of diversity with the number of supplied resources in this regime agrees with analytic predictions obtained for “typical” communities. But at low supply rates, only species that directly consume the externally supplied resource can survive, and the resulting low-diversity state no longer satisfies the assumptions for calculating “typical” community properties.</p>

<h3 id="references">References</h3>

<ol class="bibliography"><li><span id="cui2019diverse">Cui, W., Marsland III, R., &amp; Mehta, P. (Preprints). Diverse communities behave like typical random ecosystems. <i>ArXiv Preprint ArXiv:1904.02610</i>.</span>

    

    
    <a href="https://arxiv.org/abs/1904.02610"><button class="btn btnId btnPub--BibTex" id="b_cui2019diverse-url" style="outline:none;">arXiv</button></a>
    
    
    
    
    

    </li>
<li><span id="marsland2019community">Marsland III, R., Cui, W., Goldford, J., &amp; Mehta, P. (2020). The Community Simulator: A Python package for microbial ecology. <i>PLoS ONE</i>, <i>15</i>, e0230430.</span>

    
    <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230430"><button class="btn btnId btnPub--BibTex" id="b_marsland2019community-url" style="outline:none;">Journal</button></a>
    

    
    <a href="https://arxiv.org/abs/1904.09367"><button class="btn btnId btnPub--BibTex" id="b_marsland2019community-url" style="outline:none;">arXiv</button></a>
    
    
    
    <a href="/assets/repository/marsland2019community.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
    
    

    </li>
<li><span id="marsland2019minimal">Marsland III, R., Cui, W., &amp; Mehta, P. (2020). A minimal model for microbial biodiversity can reproduce experimentally observed ecological patterns. <i>Scientific Reports</i>, <i>10</i>, 3308.</span>

    
    <a href="https://www.nature.com/articles/s41598-020-60130-2"><button class="btn btnId btnPub--BibTex" id="b_marsland2019minimal-url" style="outline:none;">Journal</button></a>
    

    
    <a href="https://arxiv.org/abs/1904.12914"><button class="btn btnId btnPub--BibTex" id="b_marsland2019minimal-url" style="outline:none;">arXiv</button></a>
    
    
    
    <a href="/assets/repository/marsland2019minimal.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
    
    

    </li>
<li><span id="marsland2019available">Marsland III, R., Cui, W., Goldford, J., Sanchez, A., Korolev, K., &amp; Mehta, P. (2019). Available energy fluxes drive a transition in the diversity, stability, and functional structure of microbial communities. <i>PLoS Computational Biology</i>, <i>15</i>(2), e1006793.</span>

    
    <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006793"><button class="btn btnId btnPub--BibTex" id="b_marsland2019available-url" style="outline:none;">Journal</button></a>
    

    
    <a href="https://arxiv.org/abs/1805.12516"><button class="btn btnId btnPub--BibTex" id="b_marsland2019available-url" style="outline:none;">arXiv</button></a>
    
    
    
    <a href="/assets/repository/marsland2019available.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
    
    

    
    <a href="/assets/repository/marsland2019available.zip"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">Supplement</button></a>
    </li></ol>


  </div><a class="u-url" href="/research/typical-ecology" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Robert Marsland</h2> -->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">Robert A Marsland III</li><li><a class="u-email" href="mailto:marsland@alum.mit.edu">marsland@alum.mit.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/robertvsiii"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">robertvsiii</span></a></li><li><a href="https://www.twitter.com/marslandbobby"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#twitter"></use></svg> <span class="username">marslandbobby</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>τοῦ λόγου δ&#39; ἐόντος ξυνοῦ ζώουσιν οἱ πολλοὶ ὡς ἰδίαν ἔχοντες φρόνησιν</p>
      </div>
    </div>

  </div>

</footer></body>

</html>
